### PR TITLE
Match 3 Functions (z_actor OK, z_mir_ray OK, z_mir_ray3 OK)

### PR DESCRIPTION
**Pull request by engineer124**
- Actor_DrawLensActors
- MirRay_ReflectedBeam
- func_80B9E8D4

### CODE REVIEW

**Review by tom-overton — APPROVED** [review]

**Review by EllipticEllipsis — APPROVED** [review]

File: src/code/z_actor.c [hunk 1]
```diff
@@ -2614,200 +2614,128 @@ void Actor_DrawLensOverlay(Gfx** gfxP, s32 lensMaskSize) {
...
-void Actor_DrawLensActors(PlayState* play, s32 numActors, Actor** actors) {
+void Actor_DrawLensActors(PlayState* play, s32 numInvisibleActors, Actor** invisibleActors) {
+    s32 dbgVar1;
+    s32 dbgVar2;
     s32 spB4;
+    Actor** invisibleActor;
     Gfx* spAC;
-    void* spA8; // pad
+    void* spA8;
     s32 spA4;
```
Comment (AngheloAlf) [inline src/code/z_actor.c:2624]: Would be better to type `spA4` as a `UNK_PTR` instead?

**Review by AngheloAlf — COMMENTED** [review]
Damn scoped temps.
Good job

File: src/code/z_actor.c (hunk 1 above)
Comment (engineer124) [reply src/code/z_actor.c:2624]: looking at the play PR, `unk_18E68` seems to be a `void*`, so probably just `void*` should be fine?

Comment (EllipticEllipsis) [reply src/code/z_actor.c:2624]: It should be the same type, at least.

Comment (AngheloAlf) [reply src/code/z_actor.c]: yeah, `void*` sounds fine

**Review by AngheloAlf — APPROVED** [review]

**Pull request merged**